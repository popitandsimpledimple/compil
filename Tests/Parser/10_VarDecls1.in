program main;
var
    e: double = 1.1 + 1.2 / 1.4;
    x: integer = 1 + 2 * 3;
    y: string = '1';
    z: char = #123;
begin end.