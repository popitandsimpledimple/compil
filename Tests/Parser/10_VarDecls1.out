program
   main
   decls
      vars
         var
            e
            type
               double
               +
                  1.1
                  /
                     1.2
                     1.4
         var
            x
            type
               integer
               +
                  1
                  *
                     2
                     3
         var
            y
            type
               string
               '1'
         var
            z
            type
               char
               #123
   block
      compounds statements
